source/scss/style.scss

```diff
@@ -48,6 +49,7 @@
 @import "components/text-image";
 @import "components/tab-menu";
 @import "components/dropdown";
+@import "components/fti-select";
```

Comment:
Maybe we should rename it to `select`, since we are not using `fti` in any other component. What do you think?